api/src/main/java/org/openmrs/module/aijar/api/impl/AijarServiceImpl.java

```diff
@@ -309,10 +309,11 @@ public void stopActiveOutPatientVisits() {
 
 		SimpleDateFormat formatter = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
 
-		SimpleDateFormat formatterExt = new SimpleDateFormat("yyyy-MM-dd");
+		SimpleDateFormat formatterExt = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 
-		String formattedDate = formatterExt.format(new Date()) + " " + "00:00:00";
+		String formattedDate = formatterExt.format(OpenmrsUtil.firstSecondOfDay(new Date()));
```

Comment:
@ssmusoke check the changes that responds to one of the comments you made

Reply:
@slubwama no need for this since the database already has a similar function